firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -270,6 +260,7 @@ public static FirebaseFirestore testFirestore(
             new EmptyCredentialsProvider(),
             asyncQueue,
             /*firebaseApp=*/ null);
+    SQLitePersistence.clearPersistence(context, databaseId, persistenceKey);
```

Comment:
Use the public API for this.

In the other comment I recommended making the public API package-private. To make it accessible, add a `clearPersistence` method like `shutdown` to `AccessHelper`.